packages/sdk/src/index.ts

```diff
@@ -32,12 +32,11 @@ export { IdentityType } from "./models/identityType";
 
 export { NotificationTarget, NotificationTargetType } from "./notification/interface";
 export {
-  BotNotification,
-  BotNotificationOptions,
   Channel,
   IncomingWebhookTarget,
   Member,
   sendAdaptiveCard,
   sendMessage,
   TeamsBotInstallation,
 } from "./notification/notification";
+export { ConversationBot, ConversationOptions } from "./notification/conversation";
```

Comment:
I'm considering to rename the folder name... since it's not for `notification` only.

Reply:
This is already added to my TODO, see the PR description. I didn't rename the folder name here is for better PR review

Reply:
E.g., to `conversation/`